packages/ui-components/src/components/tooltip/tooltip.types.ts

```diff
@@ -12,4 +12,11 @@ export interface ITooltip {
 	disabled?: boolean;
 	/** (optional) Content element reference */
 	contentElement?: HTMLElement;
+	/** (optional) Display tooltip */
+	behaviour?: ETooltipBehaviour;
+}
+
+export enum ETooltipBehaviour {
+	Hover = 'hover',
```

Comment:
Sorry, but this seems a bit odd to me. Since we are not adding a "click" behaviour to the list, seems that the 'hover' and 'truncate' are the same behaviour. The difference is that we use 'truncate' the child container will be ellipsed/truncated.

Can you refactor this so the truncate becomes a boolean prop?

Sorry for the trouble.